server/service/src/sync/synchroniser.rs

```diff
@@ -279,7 +295,7 @@ impl Synchroniser {
                 false => Some(logger),
                 true => None,
             },
-            None,
+            central_sync_server_id, // Also include OMS Central as source site_id is null, should probably be passing a vec here...
```

Comment:
As per earlier comment, should we make this a vec, and have a OMS Central Site ID set too?

Reply:
The intention for this was to filter just by one site and the only requirement was for remote sites when they push to central (integration on omSupply central). Now there is this requirement i think we should go to this:

https://github.com/msupply-foundation/open-msupply/compare/8209-sync-vaccination-course...8209-sync-vaccination-course(get-source-site-id-from-sync-buffer)

Interesting enough you were on your way there in this PR: https://github.com/msupply-foundation/open-msupply/pull/4392, but my suggestion changed it to what it is now: https://github.com/msupply-foundation/open-msupply/pull/4392/commits/ba0055445b6fb0062f5487e43ebc3ee57afeb448

I think i was thinking of one use case and also wasn't super happy with mutating integration operation

Reply:
Changed to this approach, but kept the integrating null source site id records from OMS Central.
Other option is to do two integrations on central sync (one for Legacy and one for OMS Central records?)

I also wondered about a magic number for OMS Central site_id like `-1` otherwise we'll need mSupply to provide use with a site_id for OMS Central.
I think this can all be solved when moving to sync v7 though.  I think it's ok to keep OMS Central records with `null` source_site_id